lib/FileGeneration.php

```diff
@@ -105,15 +107,35 @@ public function generate(bool $regen = false)
         $packed = $initOutput;
         $jsFilePath = $site.'/'.'init.js';
         file_put_contents($jsFilePath, $packed);
+
+        $remoteUrl = $this->remote_url;
+        $remoteName = 'origin';
+
+        if (! $regen) {
+            $repo = \Cz\Git\GitRepository::init($site);
+            $repo->addRemote($remoteName, $remoteUrl);
+            $repo->addAllChanges();
+            $repo->commit(time());
+        } else {
+            $repo = new \Cz\Git\GitRepository($site);
+            if ($remoteUrl) {
+                $repo->setRemoteUrl($remoteName, $remoteUrl);
+            }
+            if ($repo->hasChanges()) {
+                $repo->addAllChanges();
+                $repo->commit(time());
+            }
+        }
+
+        $repo->push($remoteName, ['master']);
     }
 
     // https://andy-carter.com/blog/recursively-remove-a-directory-in-php
     private function cleanupDir(string $path) {
-        $files = glob($path . '/*');
+        $files = glob($path . '/*'); // glob ignores all hidden files
        foreach ($files as $file) {
            is_dir($file) ? $this->cleanupDir($file) : unlink($file);
        }
-       rmdir($path);
```

Comment:
why?

Reply:
We are now using git. So, we cannot delete the folder itself as it contains .git folder.